[2023-09-04T11:55:21.588+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v05.insert_into_table scheduled__2023-01-27T00:00:00+00:00 [queued]>
[2023-09-04T11:55:21.599+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v05.insert_into_table scheduled__2023-01-27T00:00:00+00:00 [queued]>
[2023-09-04T11:55:21.599+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T11:55:21.618+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-01-27 00:00:00+00:00
[2023-09-04T11:55:21.625+0000] {standard_task_runner.py:57} INFO - Started process 14587 to run task
[2023-09-04T11:55:21.627+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v05', 'insert_into_table', 'scheduled__2023-01-27T00:00:00+00:00', '--job-id', '2638', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpzdww1ur2']
[2023-09-04T11:55:21.630+0000] {standard_task_runner.py:85} INFO - Job 2638: Subtask insert_into_table
[2023-09-04T11:55:21.679+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v05.insert_into_table scheduled__2023-01-27T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T11:55:21.771+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v05' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-01-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-27T00:00:00+00:00'
[2023-09-04T11:55:21.772+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-01-27', 'dag_with_postgres_operator_v05')
        
[2023-09-04T11:55:21.780+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T11:55:21.788+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T11:55:21.799+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-01-27', 'dag_with_postgres_operator_v05')
        , parameters: None
[2023-09-04T11:55:21.803+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T11:55:21.817+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v05, task_id=insert_into_table, execution_date=20230127T000000, start_date=20230904T115521, end_date=20230904T115521
[2023-09-04T11:55:21.840+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T11:55:21.857+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
